app/app/Http/Controllers/ChangePasswordController.php

```diff
@@ -32,36 +33,26 @@ public function post(Request $request)
         if (BaseUser::isSignedIn())
         {
 			$user = BaseUser::getDbUser();
-			$validation_rules = array(
-				'current_password'     => 'required',
-				'new_password'         => 'required',
-				'password_confirm'     => 'required|same:new_password'
-			);
-			$validator = Validator::make(Input::all(), $validation_rules);
-			$failing = $validator->fails();
-			if (!$failing && !empty($user->password_hash))
-			{
-				if ( !BaseUser::authenticate($user->email, $request->input('current_password')) )
-				{
-					$validator->errors()->add('current_password', 'Current password not matched');
-					$failing = true;
-				}
+			$validator = Validator::make(
```

Comment:
Why not leave the $validation_rules variable unchanged?

Reply:
No Problem, I will update it